src/hypothesis/settings.py

```diff
@@ -257,8 +259,41 @@ def __exit__(self, *args, **kwargs):
         default_context_manager = self.defaults_stack().pop()
         return default_context_manager.__exit__(*args, **kwargs)
 
-    default = DefaultSettings()
+    @staticmethod
+    def register_profile(name, **kwargs):
```

Comment:
Not to mention using the actual object is more in line with the whole Zen thing `Explicit is better than implicit.` :-P